### PR TITLE
Don't show "Add changes" if project was not created by current user

### PR DESCRIPTION
**Pull request by jwalgran**
## Overview

Users can't save changes to projects they do not own so it is confusing to allow editing.

Connects #2670 

### Demo

screen shot 2018-02-22 at 8 48 28 am

## Testing Instructions

 * Log in, create a project, run TR-55, and share the project to Hydroshare.
 * Refresh the project page and verify that the "Add changes" button is visible.
 * Copy the project URL.
 * Log in as a different user and browse the project page.
 * Verify that the "Add changes" button is not visible.

### CODE REVIEW

File: src/mmw/js/src/modeling/views.js [hunk 1]
```diff
@@ -874,13 +874,15 @@ var ScenarioToolbarView = Marionette.CompositeView.extend({
         var gisData = this.currentConditions.getGisData().model_input,
             isGwlfe = this.modelPackage === coreUtils.GWLFE && !_.isEmpty(gisData),
             isOnlyCurrentConditions = this.collection.length === 1 &&
-                this.collection.first().get('is_current_conditions');
+                this.collection.first().get('is_current_conditions'),
+            createdByCurrentUser = this.model.get('user').id === App.user.get('id');
```
Comment (rajadain) [inline src/mmw/js/src/modeling/views.js]: Prefer reusing the [`isEditable`](https://github.com/WikiWatershed/model-my-watershed/blob/986f51fa7d46eb2ece2a53a8776c5cfb75ca4f77/src/mmw/js/src/modeling/views.js#L1144-L1146) method here, as it is elsewhere in this file.

**Comment by arottersman** [comment]
I noticed the "Remove Modification" buttons remain for both MapShed and TR-55. They are a lot more subtle than the "Add changes to area" button, but I think their removal is in line with this work.

screen shot 2018-02-22 at 11 26 12 am

screen shot 2018-02-22 at 11 40 42 am

**Comment by jwalgran** [comment]
>Prefer reusing the isEditable method

Thanks for the pointer. I added a fixup.

> I noticed the "Remove Modification" buttons remain for both MapShed and TR-55.

Thanks for pointing those out. I'll add a commit to hide them as well.

**Comment by rajadain** [comment]
@azavea-bot rebuild

**Comment by arottersman** [comment]
@jwalgran, is this ready for another look?

**Comment by jwalgran** [comment]
Yes. This is now ready.

**Pull request merged**